### PR TITLE
feat: add credit class details section with credibility cards

### PR DESCRIPTION
**Pull request by blushi**
## Description

Closes: regen-network/regen-registry#1643
Closes: regen-network/regen-registry#1724

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [x] provided a link to the relevant issue or specification
- [x] provided instructions on how to test
- [x] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed

### How to test

- with credibility cards: https://deploy-preview-1970--regen-registry.netlify.app/credit-classes/C02
- without: https://deploy-preview-1970--regen-registry.netlify.app/credit-classes/C01

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

**Comment by blushi** [comment]
@erikalogie see testing links above

File: web-registry/src/components/organisms/DetailsSection/DetailsSection.tsx [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+import { useSectionStyles } from './DetailsSection.styles';
+import { DetailsSectionProps } from './DetailsSection.types';
+
+export const DetailsSection: React.FC<
+  React.PropsWithChildren<DetailsSectionProps>
+> = ({
+  header,
+  credibilityCards,
+  methodology,
+  creditClassDoc,
```
Comment (blushi) [inline web-registry/src/components/organisms/DetailsSection/DetailsSection.tsx:26]: pending answer from Sam with regards to the missing credit class doc field in the metadata

**Comment by erikalogie** [comment]
This looks great @blushi! What happened to the credit class button in the end?

Also, is this paragraph in Sanity? "These credits use the most scientifically rigorous and strict criteria to ensure that every purchase is real and quantifiable."

**Comment by blushi** [comment]
> This looks great @blushi! What happened to the credit class button in the end?

As I said in the comment above https://github.com/regen-network/regen-web/pull/1970#discussion_r1262556073, I'm still waiting on Sam's answer for that

> Also, is this paragraph in Sanity? "These credits use the most scientifically rigorous and strict criteria to ensure that every purchase is real and quantifiable."

Yes, currently set up to be the same on all credit classes page similarly to the project details description on the project page. Is that fine or should this be specific to credit classes instead?

File: web-registry/src/components/organisms/DetailsSection/DetailsSection.tsx (hunk 1 above)
Comment (ryanchristo) [reply web-registry/src/components/organisms/DetailsSection/DetailsSection.tsx:26]: cc @S4mmyb

**Review by ryanchristo — COMMENTED** [review]
Looks good to me. Pending credit class button and whether we need to adjust where we store the first sentence of the credit class details section.

The checkmarks feel a bit close to the text and maybe they should be top aligned so that they line up with the first line of text? No changes required, just an observation.

Screenshot from 2023-07-17 11-38-55

**Review by wgwz — APPROVED** [review]
utACK

**Comment by erikalogie** [comment]
> > This looks great @blushi! What happened to the credit class button in the end?
> 
> As I said in the comment above [#1970 (comment)](https://github.com/regen-network/regen-web/pull/1970#discussion_r1262556073), I'm still waiting on Sam's answer for that

Ok

> 
> > Also, is this paragraph in Sanity? "These credits use the most scientifically rigorous and strict criteria to ensure that every purchase is real and quantifiable."
> 
> Yes, currently set up to be the same on all credit classes page similarly to the project details description on the project page. Is that fine or should this be specific to credit classes instead?

That's perfect how you implemented it

**Comment by erikalogie** [comment]
> Looks good to me. Pending credit class button and whether we need to adjust where we store the first sentence of the credit class details section.
> 
> The checkmarks feel a bit close to the text and maybe they should be top aligned so that they line up with the first line of text? No changes required, just an observation.

I think there could be a little bit more space (no more than 5px) and yes I agree that the checkmarks should be lined up with the first line of text
> 
> Screenshot from 2023-07-17 11-38-55

**Comment by blushi** [comment]
> > Looks good to me. Pending credit class button and whether we need to adjust where we store the first sentence of the credit class details section.
> > The checkmarks feel a bit close to the text and maybe they should be top aligned so that they line up with the first line of text? No changes required, just an observation.
> 
> I think there could be a little bit more space (no more than 5px) and yes I agree that the checkmarks should be lined up with the first line of text
> 
> > Screenshot from 2023-07-17 11-38-55

This is now fixed.

**Comment by ryanchristo** [comment]
As discussed, let's create a follow up and merge this as is for now. Thanks!

**Pull request merged**